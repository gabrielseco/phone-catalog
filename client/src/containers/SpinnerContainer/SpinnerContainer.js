import React from 'react';
import { Spinner } from 'components';
import styles from './SpinnerContainer.scss';

const SpinnerContainer = () => (
  <div className={styles.spinnerContainer}>
    <Spinner />
  </div>
);

export default SpinnerContainer;
