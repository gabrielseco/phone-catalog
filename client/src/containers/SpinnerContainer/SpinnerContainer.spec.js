import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import SpinnerContainer from './SpinnerContainer';

describe('SpinnerContainer suite', () => {
  it('should render the SpinnerContainer component', () => {
    const jsxComponent = <SpinnerContainer />;
    const component = shallow(jsxComponent);
    const tree = renderer.create(jsxComponent).toJSON();
    expect(component).toBeDefined();
    expect(tree).toMatchSnapshot();
  });
});
