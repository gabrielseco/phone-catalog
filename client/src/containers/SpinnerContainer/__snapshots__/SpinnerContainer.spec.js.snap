// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpinnerContainer suite should render the SpinnerContainer component 1`] = `
<div
  className="spinnerContainer"
>
  <div
    aria-busy="true"
    aria-live="assertive"
    className="spinner"
    role="alertdialog"
  />
</div>
`;
